keybinding.go

```diff
@@ -286,8 +286,9 @@ const (
 
 // Modifiers.
 const (
-	ModNone Modifier = Modifier(0)
-	ModAlt           = Modifier(tcell.ModAlt)
-	// ModCtrl doesn't work with keyboard keys. Use CtrlKey in Key and ModNone. This is was for mouse clicks only (tcell.v1)
-	// ModCtrl = Modifier(tcell.ModCtrl)
+	ModNone       Modifier = Modifier(0)
+	ModAlt                 = Modifier(tcell.ModAlt)
+	ModMouseAlt            = Modifier(tcell.ModAlt)
```

Comment:
Maybe we should remove the ModMouseAlt as we already have ModAlt

Reply:
I added it for consistency, but feel free to remove it ;)

Reply:
Okie